scripts/manila-share-sync.py

```diff
@@ -92,6 +93,18 @@ def __init__(self, config_file, prom_host, interval, tasks, dry_run_tasks, prom_
         self.missing_volumes = {}
         self.offline_volumes_lock = Lock()
         self.offline_volumes = {}
+        self.net_capacity_snap_reserve = self.get_net_capacity_snap_reserve(config_file)
+
+    def get_net_capacity_snap_reserve(self, config_file):
+        """Return the snapshot_reserve_percent from the config file"""
+        parser = configparser.ConfigParser()
+        try:
+            parser.read(config_file)
+            snap_percent = parser.get('default', 'netapp_volume_snapshot_reserve_percent', raw=True)
+            return int(snap_percent)
+        except:
+            print("WARN: Manila config file missing netapp_volume_snapshot_reserve_percent, setting to 50")
```

Comment:
why print when everywhere else log. is used? 
Please change to log.warning

Reply:
fixed